pkg/cluster/k8sres.go

```diff
@@ -147,14 +148,16 @@ func fillResourceList(spec acidv1.ResourceDescription, defaults acidv1.ResourceD
 	return requests, nil
 }
 
-func generateSpiloJSONConfiguration(pg *acidv1.PostgresqlParam, patroni *acidv1.Patroni, pamRoleName string, logger *logrus.Entry) string {
+func generateSpiloJSONConfiguration(pg acidv1.PostgresqlParam, patroni acidv1.Patroni, pamRoleName string, logger *logrus.Entry) string {
 	config := spiloConfiguration{}
 
 	config.Bootstrap = pgBootstrap{}
 
 	config.Bootstrap.Initdb = []interface{}{map[string]string{"auth-host": "md5"},
 		map[string]string{"auth-local": "trust"}}
 
+	config.Bootstrap.DCS.StandbyClusterCfg = patroni.StandbyCluster
```

Comment:
The way we handle this (and probably all of the `config.Bootstrap.*`) settings should be wrapped within the same logic as https://github.com/zalando/postgres-operator/pull/513/commits/3ca1a76e2a13ec2de5f9acac365809a00625e32a

Reply:
I agree on `config.Bootstrap.*` to be handled differently in updates, but it is an out of scope of this MR